app/uk/gov/hmrc/apiplatformtest/config/microserviceGlobal.scala

```diff
@@ -37,9 +37,10 @@ trait ServiceLocatorRegistration extends GlobalSettings with RunMode {
 
   override def onStart(app: Application): Unit = {
     super.onStart(app)
-    registrationEnabled match {
-      case true => slConnector.register
-      case false => Logger.warn("Registration in Service Locator is disabled")
+    if (registrationEnabled) {
```

Comment:
Not sure we should change this even if you and I might both code it this way if we were starting from new.  It's very much a personal preference.  We would frown on if(option.isdefined) and instead pattern match on some/none - not much difference.

Reply:
fair enough